### ISSUE TITLE
Support of other languages

### ISSUE THREAD

**Issue by Aniade** [Alfendi/Pokemon-Sleep-Rater#1]
Thank you very much for the bot, it's really great!

I was wondering if there are plans to make it work in other languages? If you have any ideas on how to implement it, I'm even willing to help you with that.

**Comment by Alfendi** [comment]
Hi! At the moment, other languages isn't supported, but I can definitely add it in. Which languages were you looking to add?

**Comment by Aniade** [comment]
Hello! Personally, I'm interested in being able to post screenshots in French (it's less important to me that the bot replies in French). Let me know if you need a mapping table with the names of Pokémons, skills, natures, etc.

**Comment by Alfendi** [comment]
For sure, I can definitely include this! I'd appreciate a mapping table of the French names! I unfortunately do not speak French, but I'll look into adding it as an option for the bot's response in the future. Thank you!

**Comment by Aniade** [comment]
I found a spreadsheet that contains the names of Pokémon and their natures: https://docs.google.com/spreadsheets/d/1Eo6oWs4RA5M4c0r9M8FXJniOyhpmNmrnULabkP8kbL8/edit#gid=0 To complete that, I've created an additional spreadsheet for the sub-skills: https://docs.google.com/spreadsheets/d/1xYi60QoyTHVJXuU08xr7HEHtQnkfTi5GxTgBA9SwmjY/edit?usp=sharing

**Comment by Alfendi** [comment]
Appreciate the documents! I'm currently waiting for a Raspberry Pi to arrive next week so I can port the bot over to it. I'll be updating the code as well as adding a database so I should be able to implement the French localisation then. Thank you!

**Comment by Zeyckler** [comment]
Hi! I could help you with the Spanish translation if you want ☺️

**Comment by Alfendi** [comment]
Of course! Apologies for the delay, I'm currently in the middle of the job search and haven't had time to work more on this project. I also can't submit for verification on Discord yet because the flag hasn't gone away yet. But I'm slowly making progress, will be sure to keep everyone updated!

**Comment by Aniade** [comment]
I understand, no problem ;) Thanks for the heads up!
